source/docs/advanced/features/feature-flags.md

```diff
@@ -0,0 +1,33 @@
+---
+id: feature-flags
+title: Feature Flags
+---
+
+Some modules in Front-Commerce use feature flags to enable or disable its features.
+
+## Enabling or Disabling a feature
+
+Feature flags are resolved by the [FeatureFlagLoader](https://gitlab.com/front-commerce/front-commerce/-/blob/main/src/server/modules/front-commerce/core/loaders.js#L7) which is typically overrided in the contextEnhancer of each module supporting a feature flag (e.g. [wishlist](https://gitlab.com/front-commerce/front-commerce/-/blob/main/src/server/modules/magento2/wishlist/index.js#L21), [search](https://gitlab.com/front-commerce/front-commerce/-/blob/main/src/server/modules/front-commerce/search/index.js#L41)).
```

Comment:
```suggestion
Feature flags are resolved by the [FeatureFlagLoader](https://gitlab.com/front-commerce/front-commerce/-/blob/main/src/server/modules/front-commerce/core/loaders.js#L7) which is typically overrided in the [`contextEnhancer`](/docs/reference/graphql-module-definition.html#contextEnhancer-optional) of each module supporting a feature flag (e.g. [wishlist](https://gitlab.com/front-commerce/front-commerce/-/blob/main/src/server/modules/magento2/wishlist/index.js#L21), [search](https://gitlab.com/front-commerce/front-commerce/-/blob/main/src/server/modules/front-commerce/search/index.js#L41)).
```

Reply:
Also, links should point to a specific version (e.g: tag or commit hash) instead of branch. Otherwise, they'll quickly get out of sync and become useless.